src/index.tsx

```diff
@@ -50,21 +51,21 @@ const containerStyle: React.CSSProperties = {
   zIndex: 100001,
 };
 
-const Modal: React.FC<ModalProps> = ({ children, isOpen = false, close, elementId = 'root' }) => {
+const Modal: React.FC<ModalProps> = ({ children, isOpen = false, onClick, elementId = 'root' }) => {
```

Comment:
It's hard to tell where `onClick` is targeted, so why not name it `onOverlayClick`?

Reply:
I think it's better your proposal. I changed the name of `onClick` to `onOverlayClick`. [`e08309e` (#24)](https://github.com/microcmsio/react-hooks-use-modal/pull/24/commits/e08309e63aa77efcc9a92125e7f52beabff3baac)